conditionize.flexible.jquery.js

```diff
@@ -44,7 +44,7 @@
       // Replace fields names/ids by $().val()
       for (name in inputs) {
         e = inputs[name];
-        tmp_re = new RegExp("(" + name + ")\\b" + ifNotInQuotes, 'g')
+        tmp_re = new RegExp("(" + name.replace('\[', '\\[').replace('\]', '\\]') + ")\\s" + ifNotInQuotes, 'g')
```

Comment:
as you can see here, I have changed the word boundary (`\b`) to a space boundary (`\s`) character...

So the following will not work (which I think did before):

```
inputname=='foo' 

# This now needs to be rewritten as follows
inputname == 'foo' 

(so probably need to clarify that in the Readme)
```

Reply:
Yes it worked before, now i need to find a way to add space in arraymap and array combine, more work for me , lol